glass/points.py

```diff
@@ -342,11 +342,11 @@ def uniform_positions(
 
     Yields
     ------
-    lon:
+    lon
         Columns of longitudes for the sampled points.
-    lat:
+    lat
         Columns of latitudes for the sampled points.
-    count:
+    count
```

Comment:
They shouldn't be here. Unintentionally introduced in #381.

Reply:
Are these not necessary to make the names show up as names (bold) instead of types (italic)?

Reply:
Excellent, it renders fine on RTD! 🚀

Reply:
Nope, I think it expects the type if you have them. We don't use colons anywhere else (like in the parameters).